File: test_api.js
const fetch = require('node-fetch');
const dotenv = require('dotenv');
dotenv.config();

async function run() {
  const res = await fetch(process.env.SUPABASE_URL + '/rest/v1/events?select=*', {
    headers: {
      'apikey': process.env.SUPABASE_ANON_KEY,
      'Authorization': 'Bearer ' + process.env.SUPABASE_ANON_KEY
    }
  });
  const data = await res.json();
  console.log(data);
}
run();

File: test_api.py
import os
from supabase import create_client
from dotenv import load_dotenv

load_dotenv()
supabase = create_client(os.environ["SUPABASE_URL"], os.environ["SUPABASE_SERVICE_KEY"])

res = supabase.table("events").select("*, studios(name, primary_color, logo_url, watermark_text)").execute()
print(res.data)
